specification/azureattestation/resource-manager/Microsoft.AzureAttestation/preview/2017-11-01/AzureAttestation.json

```diff
@@ -0,0 +1,444 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "AzureAttestation",
+    "description": "The service for managing azure attestation.",
+    "version": "2017-11-01"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/providers/Microsoft.AzureAttestation/operations": {
+      "get": {
+        "tags": [
+          "Operations"
+        ],
+        "operationId": "Operations_List",
+        "description": "Lists all of the available Azure attestation operations.",
+        "x-ms-examples": {
+          "Operations_List": {
+            "$ref": "./examples/Operations_List.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "List all the operations.",
+            "schema": {
+              "$ref": "#/definitions/OperationList"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.AzureAttestation/service/{serviceName}":
+    {
+      "get": {
+        "tags": [
+          "AttestationService"
+        ],
+        "operationId": "Get_AttestationService",
```

Comment:
OperationIds must be of the form `Resource_OperationName`, please take a look at rules R1003-R1007 [here](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/openapi-authoring-automated-guidelines.md) and rename operation Ids accordingly